src/components/formreporttype.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
+import { Formik, Form, Field } from "formik";
+import Categories from "./services/categories.json";
+import * as Yup from 'yup';
+import ErrorMsg from "./errormsg"
+
+const getSubCategories = (categories, categoryId) => {
+	var cats = categories.find(category => category.id === categoryId);
```

Comment:
the variable name should be `category`. `cats` seems likes there is a list of, but this should be a single object.